src/ucp/api/ucp.h

```diff
@@ -2732,6 +2764,39 @@ ucs_status_ptr_t ucp_stream_recv_nb(ucp_ep_h ep, void *buffer, size_t count,
                                     size_t *length, unsigned flags);
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief Non-blocking stream receive operation of structured data into a
+ *        user-supplied buffer.
+ *
+ * This routine receives data that is described by the local address @a buffer,
+ * size @a count object on the endpoint @a ep. The routine is non-blocking
+ * and therefore returns immediately. The receive operation is considered
+ * complete when the message is delivered to the buffer. If the receive
+ * operation cannot be started, the routine returns an error.
+ *
+ * @param [in]     ep       UCP endpoint that is used for the receive operation.
+ * @param [in]     buffer   Pointer to the buffer to receive the data to.
+ * @param [in]     count    Number of elements to receive into @a buffer.
+ * @param [out]    length   Size of the received data in bytes. The value is
+ *                          valid only if return code is UCS_OK.
+ * @note                    The amount of data received, in bytes, is always an
+ *                          integral multiple of the @a datatype size.
+ * @param [in]     flags    Flags defined in @ref ucp_stream_recv_flags_t.
```

Comment:
same as send, probably no need for flags, need to clarify what parameters are relevant

Reply:
flags are removed